admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,82 @@
+===================
+ownCloud User Types
+===================
+
+ownCloud supports six user types.
+These are:
+
+- `Anonymous`_
+- `Guest`_
+- `Standard User`_
+- `Federated User`_
+- `ownCloud Group Administrator`_
+- `ownCloud Administrator`_
+
+The following information is not an in-depth guide, but more of a high-level overview of each type.
+
+Anonymous
+---------
+
+- Is not a regular user.
+- Has access to specific content made available via public links.
+- Can be password-protected (optional, enforced, policy-enforced).
+- Can have an expiration date (optional, enforced, enforced dependent on password).
+- Has no personal space
+- Has no file ownership (ownership of uploaded/created files is directed to sharer).
+- Has no use of clients.
+- Quota is that of the sharer.
+- Permissions are those granted by the sharer for specific content, e.g., *view-only*, *edit*, and *no file listing*.
+- Can only use apps such file and viewer apps, such as PDF, and Collabora view/edit.
```

Comment:
-> Can only use apps regarding file**s** and viewer**s**, such as PDF, and Collabora view/edit.

Reply:
Neither sentence works grammatically. I'll find another option.

Reply:
Does "*Can only use file and viewer apps*" work?

Reply:
Yes, probably fine. With "files" I explicitly meant the files app but yours should fit.